### PR TITLE
Rails 4 upgrade

### PR DESCRIPTION
**Pull request by tboerger**
Just some fixes that are included for the rails 4 update at https://github.com/crowbar/barclamp-crowbar/pull/1157 for getting feedback.

### CODE REVIEW

**Comment by tboerger** [comment]
This merge is missing :)

**Comment by vuntz** [comment]
> This merge is missing :)

I'd like to keep the TODO. If/when we do some real cleanup on this barclamp, then we can remove that file.

**Comment by tboerger** [comment]
IMHO the TODO file is currently more or less useless as there is much much more to do before we can use that. And if somebody really needs that it is still in the older branches.

File: TODO [hunk 1]
```diff
@@ -1,8 +0,0 @@
-* TODO Generic RAID volume info -> Linux device mapping
```
Comment (dirkmueller) [inline TODO]: Please add it back. the rest looks fine to me fwiw, thanks!

**Comment by tboerger** [comment]
Somebody else can migrate that.

**Pull request closed without merging**